lua/py-requirements/init.lua

```diff
@@ -46,4 +46,11 @@ function M.upgrade_all()
     actions.upgrade()
 end
 
+---Display PyPI package description in floating window
+---@param opts? table
```

Comment:
I don't think we should take any opts, will use some default behavior.

Unless there was something you were using the opts for, lmk

Reply:
Mainly for specifying how to draw the border on the float, since the default is having no border. I started off with explicitly setting `border = 'rounded'`, but realized other people may prefer to use some other style.

Reply:
Ah, gotcha, that's fair, can you think of a way to document what options are allowed / where the options are ultimately used so users can more easily determine what the opts mean in this context?

Another approach which I'm leaning more towards would be to add these in the overall configuration when setup is called, something like `lsp_window`, that way the function itself has no parameters and we can re-use the style settings if we add more kinds of popup windows rather than requiring each of them to accept options.